Harvest.Web/Startup.cs

```diff
@@ -144,6 +145,7 @@ public void ConfigureServices(IServiceCollection services)
             services.Configure<SparkpostSettings>(Configuration.GetSection("SparkPost"));
             services.Configure<StorageSettings>(Configuration.GetSection("Storage"));
             services.Configure<EmailSettings>(Configuration.GetSection("Email"));
+            services.Configure<AuthenticationSettings>(Configuration.GetSection("Authentication"));

```

Comment:
We should use the already existing auth settings file from a few lines up, the one that has cas login info.